systemtest/src/main/resources/keycloak-postgres.yml

```diff
@@ -0,0 +1,54 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: keycloak-postgres
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+kind: StatefulSet
```

Comment:
just question, why statefulset? I think deployment is enough or not? :)

Reply:
I'd say that StatefulSets are typical for databases. Since there is only a single replica here it probably doesn't matter, but better to follow common practice and the Keycloak docs[1]. 

[1] https://www.keycloak.org/operator/basic-deployment